modules/security-understanding-container-validation.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-understanding.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="security-understanding-container-validation_{context}"]
+= Container security and validation
```

Comment:
Pedantic: “container security” is much larger than the scope of this document.

“Container image autenticity?” “Preventing supply chain attacks on container images”?